src/components/modal/noteModal.css

```diff
@@ -1,3 +1,38 @@
+.edit-modal {
+    width: 37%;
+    min-width: 25rem;
+    max-width: 35rem;
+    margin-top: 10rem;
+    border-radius: 8px;
+}
+
+.modal-container{
+    position: fixed;
+    top: 0;
+    right: 0;
+    left: 0;
+    bottom: 0;
+    background: var(--overlay);
+    z-index: 4000;  
```

Comment:
why this much z-index ???

Reply:
I am using react-pro-sidebar which has a greater z-index. To overlap that I had to use larger z-index value